docs/code-reviews/rejected-suggestions/ef52094_migration-finish-core-cleanup_PR166_missing-umbrella-apis.md

```
@@ -0,0 +1,18 @@
+Title: Defer adding standalone public encode/decode APIs for attributed edges
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)